addons/transformations.md

```diff
@@ -56,7 +50,7 @@ Transformation files need to be placed in the directory `$OPENHAB_CONF/transform
     var fahrenheit = transform("JS", "convert-C-to-F.js", temperature)
     ```
 
-For the sake of simple examples, the contents of the referenced files `window_esp.map` and `convert-CToF.js` were left out.
+For the sake of simple examples, the contents of the referenced files `window_esp.map` and `convert-C-to-F.js` were left out.
```

Comment:
@ThomDietrich Perhaps consider, "To keep this (these?) examples simple, the contents of the ..."